### PR TITLE
JBPM-9571 - Stunner - Service task Thumbnails are huge in Documentation

### PR DESCRIPTION
**Pull request by Josephblt**
All icons were scaled accordingly. They all share size, color scheme and file naming pattern.

**JIRA**: [JBPM-9571](https://issues.redhat.com/browse/JBPM-9571)

**Business Central**: [WAR file](https://drive.google.com/file/d/1OCbSD8x0QtmDxHyPbLd8PZGFvVuOZaLb/view?usp=sharing)

**VS Code**: [plugin](https://drive.google.com/file/d/1fTNCx6pUFmgO0lnsH3smJ_sxO8Ie9Lhm/view?usp=sharing)

### CODE REVIEW

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by Josephblt** [comment]
Jenkins run cdb

**Comment by Josephblt** [comment]
Jenkins retest this

**Comment by Josephblt** [comment]
Hello @LuboTerifaj, @romartin. This is ready for review. Thanks.

**Comment by romartin** [comment]
I'll let @hasys own this review with @LuboTerifaj guys :+1:

**Comment by romartin** [comment]
@hasys @Josephblt although I was expecting also some HTML / CSS change here, targeting the documentation tab.
I mean, we can reduce the default icons size, that's ok, but we can't expect icons from end users to be on the right size... so I think we should also fix the generated documentation HTML stuff to scale properly all icons, either they're using the right size or not.
WDYT?

**Comment by hasys** [comment]
Hi @Josephblt, can't login to the provided business central war file. Is it on my side or war file is not complete? At least it 30 Mb less than master snapshot.

**Comment by Josephblt** [comment]
Jenkins retest this

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by Josephblt** [comment]
Jenkins run cdb

**Comment by Josephblt** [comment]
Jenkins run cdb

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by Josephblt** [comment]
Hello @LuboTerifaj, @hasys. All updated here. Ready to test. I added the .vsix but this has no effect for Kogito. So please if you could give this some priority, that would be great. Thanks.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/documentation/ClientBPMNDocumentationService.java [hunk 1]
```diff
@@ -571,8 +571,8 @@ public String getCategoryIcon(String category) {
                             .map(data -> createImageTag(data)).orElse(""));
         }
 
-        private String createImageTag(String data) {
-            return "<img src=\"" + data + "\">";
+        protected String createImageTag(String data) {
+            return "<img src=\"" + data + "\" style=\"width: " + ICON_WIDTH + "px; height: " + ICON_HEIGHT +"px\";>";
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/documentation/ClientBPMNDocumentationService.java:575]: ok now looks better - we're also forcing scaling, good, thanks @Josephblt 
Anyway, you know if this way to scale via CSS is preserving the scale ratio? I mean, is the image once scaled being enlarged horizontally or vertically?

Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/documentation/ClientBPMNDocumentationService.java:575]: This does not keep scale ratio. It just scales the image to 20x20. As for the default icons I resized them manually. I kept them 16x16 as they were originally, but scaled the canvas size. So basically we have 16x16 images inside a 20x20 canvas.

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Review by LuboTerifaj — APPROVED** [review]
Hello @Josephblt ,

the fix works great!

There are issues only with showing icons, that are not part of this jira. I have updated already existing issue: [JBPM-9494](https://issues.redhat.com/browse/JBPM-9494) _Custom task thumbnails are the same after updating WIDs_

Thank you!

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Review by romartin — APPROVED** [review]

**Comment by Josephblt** [comment]
Hello @hasys. If you could please review this that would be great. Thanks.

**Comment by romartin** [comment]
hey @hasys 
Do you want to contribute on the review? If so, please provide feedback.  Otherwise please lemme know so we can move this PR forward. Thanks!

**Comment by romartin** [comment]
@hasys is unresponsive.. merging guys!

**Comment by romartin** [comment]
Please @Josephblt let's go ahead now with the cherry-pick for `7.48.x`. Thanks in advance!

**Pull request merged**